ringbuffer.py

```diff
@@ -220,9 +252,9 @@ def try_read(self, reader):
                 order to wait for new data to arrive.
         """
         with self.lock.for_read():
-            return self._try_read_no_lock(reader)
+            return self._try_read_no_lock(reader, length=length)
```

Comment:
I believe the `Returns` line in the doc block above is now incorrect. This returns a ctypes array.